src/nni_manager/training_service/kubeflow/kubeflowTrainingService.ts

```diff
@@ -570,15 +622,32 @@ class KubeflowTrainingService implements TrainingService {
         if(this.kubeflowTrialConfig) {
             switch(roleType) {
                 case 'ps':
-                    if(this.kubeflowTrialConfig.ps && this.kubeflowTrialConfig.ps.gpuNum == 0) {
-                        runScriptLines.push(`export CUDA_VISIBLE_DEVICES=''`);
+                    if(this.kubeflowTrialConfig instanceof KubeflowTrialConfigTensorflow){
+                        if(this.kubeflowTrialConfig.ps && this.kubeflowTrialConfig.ps.gpuNum == 0) {
```

Comment:
function ```genereateRunScript``` 's name is typoed. Please help to correct it to ```generateRunScript```

Reply:
updated to generateRunScript